src/discussions/common/HoverCard.jsx

```diff
@@ -33,6 +33,7 @@ const HoverCard = ({
   const { enableInContextSidebar } = useContext(DiscussionContext);
   const { isClosed } = useContext(PostCommentsContext);
   const isUserPrivilegedInPostingRestriction = useUserPostingEnabled();
+  const hasLikePermission = useHasLikePermission(contentType, id);
```

Comment:
instead of hasLikePermission we can use isUserHasLikePermission

Reply:
I believe both naming convention represent same purpose hasLikePermission also works well gere